OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 1

# Description
Lint fix for Reports component, this is phase 1.

## Files changed

- BadgeSummaryPreview.jsx
- Badgesummaryviz.jsx  
- InfringementsViz.jsx
- PeopleTable.js

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Go to Reports > People > select a user > click "Show Badge",  Check functionality is the same as before. (Please record a video)
- Check other functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

Optional:
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- `cd src/components/Reports`
  - Run `npx eslint `for the files: BadgeSummaryPreview.jsx, BadgesummaryViz.jsx, InfringementsViz.jsx, PeopleTable.js
  - Check if you find any errors
I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!

Screenshot:
<img width="1177" alt="CleanShot 2023-09-27 at 02 47 27@2x" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/699e5203-62e8-4af8-84f5-e63fb64bc1f3">


## Note:
Include the information the reviewers need to know.
